hub/core/tests/test_compression.py

```diff
@@ -31,6 +31,9 @@
 image_compressions.remove("wmf")
 image_compressions.remove("apng")
 image_compressions.remove("dcm")
+for c in image_compressions:
+    if is_readonly_compression(c) == c:
```

Comment:
`is_readonly_compression` returns a bool, this logic is wrong.

Reply:
It is returning c.